pcap-rpcap.c

```diff
@@ -45,6 +45,10 @@
 #include "pcap-rpcap.h"
 #include "pcap-rpcap-int.h"
 
+#if defined(_MSC_VER) && !defined(strdup)
```

Comment:
And, again, `portability.h` should handle this.

Reply:
Corrects the warning.

Reply:
Again, why must this be handled in several places in the source rather than by a change to `portability.h`?